spec/schemas/_plugins._common.yaml

```diff
@@ -0,0 +1,78 @@
+openapi: 3.1.0
+info:
+  title: Schemas of _plugins._common category.
+  description: Schemas of _plugins._common category.
+  version: 1.0.0
+paths: {}
+components:
+  schemas:
+    Query:
```

Comment:
Are these common to all plugins or just SQL? Feels like sql, so should it go to `_plugins/sql/_common.yaml`?

Reply:
I got this, you're right! I fixed it